diff --git a/dist/module/lib/webauthn.dom.d.ts b/dist/module/lib/webauthn.dom.d.ts
--- a/dist/module/lib/webauthn.dom.d.ts
+++ b/dist/module/lib/webauthn.dom.d.ts
@@ -498,7 +498,7 @@
  * @see {@link https://w3c.github.io/webauthn/#publickeycredential W3C WebAuthn Spec - PublicKeyCredential}
  * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/PublicKeyCredential MDN - PublicKeyCredential}
  */
-export interface PublicKeyCredentialFuture<T extends PublicKeyCredentialJSON = PublicKeyCredentialJSON> extends PublicKeyCredential {
+export interface PublicKeyCredentialFuture<T extends PublicKeyCredentialJSON = PublicKeyCredentialJSON> {
     /**
      * The type of the credential (always "public-key").
      * @see {@link https://w3c.github.io/webauthn/#dom-credential-type W3C - type}
